### ISSUE TITLE
Box2d bipedal walker segfault

### ISSUE THREAD

**Issue by justheuristic** [openai/gym#114]
This code happens to cause segfault for some reasons i cannot comprehend

```
env_name = "BipedalWalker-v1"
import gym
ll0 = gym.make(env_name)
ll0.reset()
ll1 = gym.make(env_name)
ll1.reset()
ll0.reset()
```

However, doing the same thing with a different box2d env works fine
```
env_name = "LunarLander-v1"
import gym
ll0 = gym.make(env_name)
ll0.reset()
ll1 = gym.make(env_name)
ll1.reset()
ll0.reset()
```

Can anyone reproduce the error? Is there anything i can do to fix it?

**Comment by olegklimov** [comment]
I am able to reproduce everything you wrote. First one crashes, second doesn't. And it is not related with program termination. Scratching my head for now.

**Comment by olegklimov** [comment]
https://github.com/openai/gym/pull/118

**Comment by jietang** [comment]
Fix merged.

**Comment by justheuristic** [comment]
Guys, sorry, but could someone please assert that the fix works at ubuntu 14.04? 
I didn't manage to bypass error for `BipedalWalker-v2` at current version

**Comment by justheuristic** [comment]
@jietang
Could you please reopen
